crates/core/src/test_utils/deploy_helpers.rs

```diff
@@ -194,8 +194,24 @@ pub fn create_raw_ethereum_tx(
         data.extend_from_slice(&arg_bytes);
     }
 
-    // Create a transaction object
-    let transaction = to_kakarot_transaction(nonce, TransactionKind::Call(to), data.into());
+    let transaction = to_kakarot_transaction(nonce, TransactionKind::Call(to), Default::default(), data.into());
+    let signature =
+        sign_message(eoa_secret_key, transaction.signature_hash()).expect("Signing of ethereum transaction failed.");
+
+    let signed_transaction = TransactionSigned::from_transaction_and_signature(transaction, signature);
+    let mut raw_tx = BytesMut::new(); // Create a new empty buffer
+
+    signed_transaction.encode_enveloped(&mut raw_tx); // Encode the transaction into the buffer
+
+    raw_tx.to_vec().into()
+}
+
+/// Constructs and signs a raw Ethereum transaction based on given parameters.
+///
+/// This function creates a transaction which will transfer a certain amount of wei to a recepeint
+/// eoa. The transaction is signed using the provided EOA secret.
+pub fn create_send_eth_raw_ethereum_tx(eoa_secret_key: H256, to: Address, value: u128, nonce: u64) -> Bytes {
```

Comment:
There is a already a [helper function](https://github.com/kkrt-labs/kakarot-rpc/blob/15d2571201c4d88ee894a513942d1d1177a21d39/crates/core/src/test_utils/deploy_helpers.rs#L183) for that, I have made one specific for constructing Ethereum transactions just for sending eth, so these transactions can be created very easily without spending much brainpower.

Reply:
yeah that's what I meant, refactor the helper function you linked to be able to choose between the two cases
but otherwise, fine by me,

we could rename this specific util as `create_eth_transfer_tx` to make it crystal clear that it's simply an eth transfer

Reply:
Oh, so I tried refactoring that helper function, but at the end I could see it just being more confusing and lengthy then just having a specific function for it, and since there are only two cases possible, I don't think we would have to add more utils in future!

Changing the name to `create_eth_transfer_tx`, that sounds good!

Reply:
Done with the above change.